src/core/core.cpp

```diff
@@ -186,10 +200,10 @@ void Core::screenShot(bool first)
     {
     case Core::FullScreen:
     {
-        const QList<QScreen *> screens = qApp->screens();
-        const QDesktopWidget *desktop = QApplication::desktop();
-        const int screenNum = desktop->screenNumber(QCursor::pos());
-        *_pixelMap = screens[screenNum]->grabWindow(desktop->winId());
+        auto screen = QGuiApplication::screenAt(QCursor::pos());
```

Comment:
@tsujan It was introduced in Qt 5.10. https://doc.qt.io/qt-5/qguiapplication.html#screenAt

Reply:
@luis-pereira Thanks!

IMHO, it's time to change the min. Qt version; otherwise, development will be complicated soon.

Reply:
I'm Ok with that. 
Actually ```QGuiApplication::screenAt(QCursor::pos())``` will also be needed in lxqt-session.

Reply:
Very good. Qt 5.10 may be enough for now.